test/abstract/FlowBasicTest.sol

```diff
@@ -23,12 +23,12 @@ abstract contract FlowBasicTest is FlowUtilsAbstractTest, InterpreterMockTest {
     using Address for address;
 
     CloneFactory internal immutable iCloneFactory;
-    IFlowV5 internal immutable iFlowImplementation;
+    address internal iFlowImplementation;
```

Comment:
this should either still be immutable, or the name changed to not have the i prefix